airflow/www/templates/appbuilder/navbar_right.html

```diff
@@ -83,7 +83,12 @@
         <li><a href="{{user_profile_url}}"><span class="material-icons">account_circle</span>{{_("Your Profile")}}</a></li>
         <li role="separator" class="divider"></li>
       {% endif %}
-      <li><a href="{{auth_manager.get_url_logout()}}"><span class="material-icons">exit_to_app</span>{{_("Log Out")}}</a></li>
+      <li class="dropdown">
+        <form method="POST" action="{{auth_manager.get_url_logout()}}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        <button type="submit" class="btn btn-link"><span class="material-icons">exit_to_app</span>{{_("Log Out")}}</button>
```

Comment:
I'd be happy for help with alignment and color (my CSS abilities are good as my french 😅):

![image](https://github.com/apache/airflow/assets/60007259/af5be93b-3c5f-42b2-8530-bbbdbfdc4ba7)

Reply:
A few css suggestions:

- Make the submit button a link:
 ```<a
          href="javascript:{}"
          onclick="document.getElementById('logout').submit();" // add the id to the form
          ><span class="material-icons">exit_to_app</span>{{_("Log Out")}}</a
        >```
- You'll need to match the styles from `.dropdown-menu > li > a > .material-icons` in material-icons.css and from `.dropdown-menu > li > a` in `bootstrap-theme.css`